schema/project-level/codelists/costCategory.csv

```diff
@@ -0,0 +1,7 @@
+Code,Title,Description
+acquisition,Acquisition costs,"All payments or considerations required to acquire/lease/purchase the land, property or existing asset, and all other expenses associated with the acquisition, excluding physical construction."
+construction,Construction costs,"Expenditures incurred as a direct result of construction including labour, materials, plant, equipment, site and head office overheads and profits as well as taxes and levies. They are the total price payable for all permanent and temporary works normally included in construction contracts, including goods or materials supplied by the public authority for the supplier to install."
+renewal,Renewal costs,"The costs of replacing an asset and/or major components once they reach the end of their life, and which the public authority decides are to be included in the capital rather than the revenue budget."
+operation,Operation costs,"Costs incurred in running and managing an asset during occupation, including administrative support services, rent, insurances, energy and other environmental/regulatory inspection costs, taxes and charges."
+maintenance,Maintenance costs,"The total cost of labour, material and other related costs to retain an asset or its parts so that it can perform its required functions (ISO 15686-5. Maintenance includes conducting corrective, responsive and preventative maintenance on an asset or its parts and all associated management, cleaning, services, repainting, repairing or replacing of parts as needed for the asset to be used for its intended purpose. It does not include renewal costs."
```

Comment:
```suggestion
maintenance,Maintenance costs,"The total cost of labor, material and other related costs to retain an asset or its parts so that it can perform its required functions (ISO 15686-5. Maintenance includes conducting corrective, responsive and preventative maintenance on an asset or its parts and all associated management, cleaning, services, repainting, repairing or replacing of parts as needed for the asset to be used for its intended purpose. It does not include renewal costs."
```